### DISCUSSION TITLE
请问作者是否有意愿发布修改过的SaintCoinach？

### DISCUSSION THREAD

**Discussion by ykkykiykyk** [GpointChen/FFXIVChnTextPatch-GP#136] [Q&A] [answered]
我想在别人汉化过的档案上进行部分修改和完善，所以需要用SaintCoinach导出带offset的rawexd，如果作者可以发布就太感谢了💐

**Comment by GpointChen** [comment, marked as answer]
ok，找時間整理好發。

**Reply by GpointChen** [reply]
https://github.com/GpointChen/SaintCoinach/tree/hexcode
